app/importers/file_importers/attendance_importer.rb

```diff
@@ -12,7 +12,7 @@ def import
   end
 
   def remote_file_name
-    'attendance_export.txt'
+    ENV.fetch('FILENAME_FOR_ATTENDANCE_IMPORT')
```

Comment:
So one thing I think we'll want is "what do I need to change per-district"?  This makes this hard to do - you have to look through all the env variables.

I'd suggest pulling all this config out to a standalone class in lib, something like `DistrictConfig` that has various methods for these things?  That class can then pull those values from environment variables, from database tables, just hard code them in Ruby based on the deployment, whatever.  And we have all that written in one place and the app code doesn't need to change.

The other thing it seems like it might be useful for these kinds of things is an environment variable that's a single key saying "what Student Insights instance am i?" and is set to `somerville` or `new-bedford` or whatever.

Reply:
@kevinrobinson Really good feedback and ideas here! Right now we have `ENV['DISTRICT_NAME]` which is used in the login page and the "create new Heroku instance for new district" script.

Reply:
@kevinrobinson Maybe we want `ENV['DISTRICT_NAME]` for presentation and `ENV['DISTRICT_KEY]` for use in code.

Reply:
👍 Yeah I think the key is the most important (eg, `DISTRICT_KEY`).  And then there are a pile of other things that branch and are different based on that - some right now are just written in code specific to Somerville (like this PR), others are in ENV variables, etc.